### ISSUE TITLE
A11Y: Dropdown component missing aria-label

### ISSUE THREAD

**Issue by alisonemiller** [carbon-design-system/carbon-components-react#593]
## Detailed description

I receive the following violation against the `Dropdown` component when I run the DAP plugin.

<img width="1991" alt="carbon-a11y" src="https://user-images.githubusercontent.com/17085131/35987980-48f9e28c-0ccb-11e8-9d9b-7cd0e0c402b5.png">

> Is this issue related to a specific component?

`Dropdown` component

> What did you expect to happen? What happened instead? What would you like to see changed?

It looks like this line is missing an `aria-label`: https://github.com/carbon-design-system/carbon-components-react/blob/v5.17.8/src/components/Dropdown/Dropdown.js#L136

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

`v5.17.8`

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

**Comment by marijohannessen** [comment]
Thanks for creating this issue! We're doing a huge a11y audit this sprint, so I'm tracking this in a new issue here #609 .
